app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      before_action :authenticate_user!
+
+      def create
+        rent = Rent.create(rent_params)
+        render json: rent
+      end
+
+      def index
+        render json: Rent.all, each_serializer: RentSerializer
+      end
+
+      private
+
+      def rent_params
+        params.permit(:init_date, :return_date, :user_id, :book_id)
```

Comment:
Maybe, could be usefull to `require` required params. Also could be usefull to rescue error and have some method to render then.
We use to used `rescue_from` and define them in the apicontroller